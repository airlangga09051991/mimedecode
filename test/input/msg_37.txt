From test Sat Feb  1 00:00:00 2014
Content-Type: multipart/mixed; boundary=ABCDE

--ABCDE
Content-Type: text/x-one

Blah

--ABCDE
--ABCDE
Content-Type: text/x-two

Blah

--ABCDE
--ABCDE
--ABCDE
--ABCDE
Content-Type: text/x-two

Blah

--ABCDE--
