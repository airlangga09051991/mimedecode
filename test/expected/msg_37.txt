From test Sat Feb  1 00:00:00 2014
Content-Type: multipart/mixed; boundary=ABCDE

--ABCDE
Content-Type: text/x-one

Blah

--ABCDE
Content-Type: text/x-two

Blah

--ABCDE
Content-Type: text/x-two

Blah

--ABCDE--
